lightning_server.cc

```diff
@@ -15,7 +15,15 @@ LightningServer::LightningServer(const char* file_name)
   config_parser_.Parse(file_name, &config_);
   std::cout << config_.ToString() << std::endl;
   ServerConfig ConfigWrapper(config_);
```

Comment:
Fixed in [this commit](https://github.com/UCLA-CS130/Mr.-Robot-et-al./pull/37/commits/5cb8eb26f24ac510f72d57d987b4e2f6d7438f06)!